source/common/common/BUILD

```diff
@@ -174,6 +176,19 @@ envoy_cc_library(
     ],
 )
 
+envoy_cc_library(
+    name = "regex_lib",
+    srcs = ["regex.cc"],
+    hdrs = ["regex.h"],
+    deps = [
+        ":assert_lib",
+        "//include/envoy/common:regex_interface",
+        "//source/common/protobuf:utility_lib",
+        "@com_googlesource_code_re2//:re2",
```

Comment:
Do you want to reference this via the external deps field in `envoy_cc_library`?

Reply:
I thought that was the old way and this is the new way via some comments I found in the bzl files. No?

Reply:
Ah, I think you are right. Never mind.